.eslintrc.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+    'root': true,
+    'env': {
+        'node': true
+    },
+    'extends': [
+        'eslint:recommended',
+        'plugin:unicorn/recommended',
+    ],
+    'parser': '@typescript-eslint/parser',
+    'parserOptions': {
+        'ecmaVersion': 2017,
+        'sourceType': 'module',
+        'ecmaFeatures': {
+            'jsx': false,
+        },
+        'project': 'tsconfig.json',
+    },
+    'plugins': [
+        'unicorn',
+        'import',
+        '@typescript-eslint/tslint',
+    ],
+    'rules': {
```

Comment:
Are you sure these are the rules currently in tslint? Please elaborate on why these rules need to be specified here.